src/core/packet_builder.c

```diff
@@ -314,7 +314,7 @@ QuicPacketBuilderPrepare(
             Builder->MinimumDatagramLength =
                 MaxUdpPayloadSizeForFamily(
                     QuicAddrGetFamily(&Builder->Path->RemoteAddress),
-                    QUIC_INITIAL_PACKET_LENGTH);
+                    Builder->Path->Mtu);
```

Comment:
Why change this?

Reply:
This is more correct. Min MTU is something we now allow to be set by settings, which also becomes a way for us to set minimum packet size.

Reply:
Min MTU can never be less than `QUIC_INITIAL_PACKET_LENGTH` right?

Reply:
No, it can never be less, but it can be more. And if set to more, we want this to be that higher value on the client side to enable a higher amplification window.